### PR TITLE
config: make pd and tikv use github permission

### PR DESCRIPTION
**Pull request by hi-rustin**
ref: https://github.com/tikv/community/pull/122

### CODE REVIEW

**Comment by hi-rustin** [comment]
cc: @BusyJay @nolouch

Could you please take a look?

**Review by Mini256 — APPROVED** [review]

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -130,26 +112,9 @@ ti-community-owners:
     require_lgtm_label_prefix: require-LGT
   - repos:
       - tikv/tikv
-    trusted_teams:
-      - maintainers
+    use_github_permission: true
     sig_endpoint: https://bots.tidb.io/ti-community-bot
     require_lgtm_label_prefix: require-LGT
-    branches:
```
Comment (BusyJay) [inline prow/config/external_plugins_config.yaml:137]: What does these mean?

Comment (hi-rustin) [reply prow/config/external_plugins_config.yaml:137]: These configurations were used to add permissions to the qa-release-merge team for the release branch, now they have been added as committers so they are no longer needed.

Comment (BusyJay) [reply prow/config/external_plugins_config.yaml:137]: Does code freeze still work?

Comment (hi-rustin) [reply prow/config/external_plugins_config.yaml:137]: > Does code freeze still work?

Yes, it also works, and code freezing is achieved by controlling code merging through labels. It has nothing to do with permissions.

**Review by BusyJay — APPROVED** [review]

**Comment by hi-rustin** [comment]
@BusyJay Thanks for your review!

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**